index.js

```diff
@@ -4,11 +4,13 @@
 /* eslint-disable no-unused-vars */
 import {AppRegistry, Platform} from 'react-native';
 
-import Mattermost from 'app/mattermost';
+import Mattermost from './app/mattermost';
 import ShareExtension from 'share_extension/android';
 
 if (Platform.OS === 'android') {
     AppRegistry.registerComponent('MattermostShare', () => ShareExtension);
 }
 
-const app = new Mattermost();
+if (Platform.OS === 'ios') {
+    const app = new Mattermost();
+}
```

Comment:
I'm conflicted by this.   The PR addresses cold startup time for Android yet it makes core changes that would affect both Android and iOS.

The changes made are iOS compliant for the most part some UI cosmetic changes.  My concern is around push notification logic.  The logic has been ported yet it would need throughout testing to ensure there's no regression.

Since it would require creating a release build I decided just to keep original functionality for iOS.

We'll look at a proper solution for iOS in the next couple of days